tools/cgconvert/CGConvert.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * File: CGConvert.cpp
+ * License: Part of the MetaCG project. Licensed under BSD 3 clause license. See LICENSE.txt file at
+ * https://github.com/tudasc/metacg/LICENSE.txt
+ */
+
+#include "config.h"
+
+#include <iostream>
+#include <fstream>
+
+#include "LoggerUtil.h"
+#include "io/MCGReader.h"
+#include "io/VersionTwoMCGWriter.h"
+
+// This may appear to be unused, but the variables declared here have side effects on the graph lib
+#include "metadata/BuiltinMD.h"
+
+using namespace metacg;
+
+int main(int argc, char** argv) {
+  auto console = MCGLogger::instance().getConsole();
+  auto errConsole = MCGLogger::instance().getErrConsole();
+
+  console->info("Running metacg::CGConvert (version {}.{})\nGit revision: {}", MetaCG_VERSION_MAJOR,
+                MetaCG_VERSION_MINOR, MetaCG_GIT_SHA);
+
+  if (argc < 3) {
+    errConsole->error("Usage: {} <input_file> <output_file> [output_version]", argv[0]);
+    return EXIT_FAILURE;
+  }
+
+  std::string inputFile = argv[1];
```

Comment:
For arguments, I'd prefer if we can transition to something like `cxxopts`, which is used in PGIS already.